ironfish/src/fileStores/config.ts

```diff
@@ -330,6 +336,7 @@ export class Config extends KeyStore<ConfigOptions> {
       jsonLogs: false,
       explorerBlocksUrl: DEFAULT_EXPLORER_BLOCKS_URL,
       explorerTransactionsUrl: DEFAULT_EXPLORER_TRANSACTIONS_URL,
+      authToken: DEFAULT_TCP_AUTH_TOKEN,
```

Comment:
We should name this `rpcAuthToken`

Reply:
The name of this default doesn't match the config.

Reply:
Done.